zellij-tile/src/data.rs

```diff
@@ -71,8 +105,7 @@ impl Default for InputMode {
 #[derive(Default, Debug, Clone, Serialize, Deserialize)]
 pub struct ModeInfo {
     pub mode: InputMode,
-    // FIXME: This should probably return Keys and Actions, then sort out strings plugin-side
-    pub keybinds: Vec<(String, String)>, // <shortcut> => <shortcut description>
+    pub keybinds: Vec<(Key, Vec<Action>)>,
```

Comment:
This looks awesome! You said the HashMap had some issues with JSON, right?

Reply:
Yes, Json can not serialize `HashMap<Key, ..>` because Key will be serialized as a map but a map can not be a key in json.